pkg/chunk/elastic/es_index_client_test.go

```diff
@@ -0,0 +1,56 @@
+package elastic
+
+// There's no good embedded ElasticSearch, so we use a real ElasticSearch instance.
```

Comment:
There's no embedded ES and this header is a great summary of how to get a local ES in docker. However, these tests can be uncommented but protected from running normally using a build flag. There are other integration tests that work against sql using `// +build integration`, so that does offer an example of how to do it for the ES tests. I would chose a different flag however, since Elasticsearch isn't (for at least now) likely to be part of the core integration tests.